stdlib/string.ncl

```diff
@@ -334,6 +342,11 @@
       Note that this function may perform better by sharing its partial application between multiple calls,
       because in this case the underlying regular expression will only be compiled once (See the documentation
       of `string.is_match` for more details).
+
+      WARNING: this function will currently return incorrect indices if `str`
```

Comment:
Ah, this one was also missing. Adding it as part of the reformatting

Reply:
Yeah, I noticed that one when rebasing #1196. It's fixed there as well.